.connector-store/.connector-store/meta.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "Adobe Campaign Classic",
+  "owner": "WSO2",
+  "category": "Developer Tools",
```

Comment:
Does this come under Developer Tools? Shall we rename it to suitable category? Maybe "Marketing/Social Media"

Reply:
As we discussed offline, the Category changed to `Sales and Marketing`.

Changed with [45a4639](https://github.com/wso2-extensions/esb-connector-adobecampaignclassic/pull/1/commits/45a46394817c0dde53960be37bd29ea97ba52435).